hedera-node/src/main/java/com/hedera/services/txns/contract/ContractCreateTransitionLogic.java

```diff
@@ -122,30 +122,23 @@ public void doStateTransition() {
 				txnCtx.consensusTime()
 		);
 
-		HederaAccountCustomizer customizer = null;
+		var transactionBody = txnCtx.accessor().getTxn().getContractCreateInstance();
+		var autoRenewPeriod = transactionBody.getAutoRenewPeriod();
```

Comment:
You can do `op.getAutoRenewPeriod().getSeconds()` instead of getting the GRPC tx again. It is retrieved at the start of the method